### PR TITLE
Converts station bounced radio to shortwave radio

### PR DESCRIPTION
**Pull request by UndeadNappist**
Name changes, is all.

### CODE REVIEW

File: html/archivedchangelog.html [hunk 1]
```diff
@@ -4131,7 +4131,7 @@ <h2 class="date">August 1, 2012</h2>
 	<h3 class="author">Sieve updated:</h3>
 	<ul class="changes bgimages16">
 		<li class="rscadd">Borgs can now have an encryption key installed into their internal radios. Simply ID, open the panel, and use the key to insert it (Screwdriver to remove)</li>
-		<li class="rscadd">Due to that as well, borgs have a 'Toggle Broadcast Mode' button for their radios, which changes the broadcast type between station-bounced (Non-reliant on TComms), and subspace (Required for department channels)</li>
+		<li class="rscadd">Due to that as well, borgs have a 'Toggle Broadcast Mode' button for their radios, which changes the broadcast type between shortwave radio (Non-reliant on TComms), and subspace (Required for department channels)</li>
```
Comment (AndrewL97) [inline html/archivedchangelog.html]: dont update changelogs

File: html/archivedchangelog.html [hunk 2]
```diff
@@ -4311,7 +4311,7 @@ <h3 class="author">Giacom updated:</h3>
...
 		<li class="rscadd">You can show the AI the notes on your PDA by holding it up to a camera. When you show up a paper/pda to the camera the AI can now click on your name to go to you, if you're near a camera. People who are Unknown will not have a link; which would've allowed the AI to track them.</li>
 		<li class="rscadd">Made the" common server" and the "preset right receiver" listen for frequencies 144.1 to 148.9. This will allow people to use different frequencies to talk to eachother without bothering the common channel. It will also allow Revs and Cultists to work with each other; everything is still logged though so it still has risks.</li>
-		<li class="rscadd">Increased the maximum frequency limit for handheld radios and intercoms. It will give you the option to just use station bounced radios on a higher frequency so that anyone with a headset can't simply tune in.</li>
+		<li class="rscadd">Increased the maximum frequency limit for handheld radios and intercoms. It will give you the option to just use shortwave radios on a higher frequency so that anyone with a headset can't simply tune in.</li>
```
Comment (AndrewL97) [inline html/archivedchangelog.html]: dont update changelogs

File: html/archivedchangelog.html (hunk 1 above)
Comment (UndeadNappist) [reply html/archivedchangelog.html]: oops

File: html/archivedchangelog.html [hunk 3]
```diff
@@ -4131,7 +4131,7 @@ <h2 class="date">August 1, 2012</h2>
 	<h3 class="author">Sieve updated:</h3>
 	<ul class="changes bgimages16">
 		<li class="rscadd">Borgs can now have an encryption key installed into their internal radios. Simply ID, open the panel, and use the key to insert it (Screwdriver to remove)</li>
-		<li class="rscadd">Due to that as well, borgs have a 'Toggle Broadcast Mode' button for their radios, which changes the broadcast type between station-bounced (Non-reliant on TComms), and subspace (Required for department channels)</li>
+		<li class="rscadd">Due to that as well, borgs have a 'Toggle Broadcast Mode' button for their radios, which changes the broadcast type between station-bounced radio (Non-reliant on TComms), and subspace (Required for department channels)</li>
```
Comment (AndrewL97) [inline html/archivedchangelog.html]: stop being such a goosey goose

**Pull request merged**